migration-engine/core/tests/test_harness/command_helpers.rs

```diff
@@ -9,6 +10,16 @@ pub struct InferAndApplyOutput {
     pub migration_output: MigrationStepsResultOutput,
 }
 
+impl InferAndApplyOutput {
+    pub fn sql_migration(&self) -> Vec<SqlMigrationStep> {
+        let mut steps = self.migration_output.database_steps.clone();
+        steps.as_array_mut().unwrap().iter_mut().for_each(|value| {
+            value.as_object_mut().unwrap().remove("raw");
+        });
```

Comment:
We can make this nicer (also where we serialize it) with a custom struct, but I wanted to touch as little code as possible to avoid making the PR bigger than it already is.